tooltester/README.md

```diff
@@ -47,65 +80,84 @@ Now, run
 
 To determine the cluster name for each context you do the following on the AWS web console:
 
 1. Go to the "Batch" [section](https://us-east-1.console.aws.amazon.com/batch) of the AWS Console
 2. Go to the "Compute Environment" [menu](https://us-east-1.console.aws.amazon.com/batch/home?region=us-east-1#console-settings/compute-environments) of the Batch section of the AWS console.
 3. Find the Batch Compute Environment associated with your context. If you click on each compute environment and look under tags, you will see the name you chose for the context when you set it up. As an example, the batch compute environment for one of my contexts is `BatchTaskBatchComputeEnv-0FxjUpJuXVowThgf`.
-4. Go to the "ECS" section of the AWS console
-5. Go to the "Clusters" [menu](https://us-east-1.console.aws.amazon.com/ecs/v2/clusters?region=us-east-1)
-6. Find the ECS cluster that starts with your Batch Computer Environment that you identified before. For example, in my case, the ECS cluster is called, `BatchTaskBatchComputeEnv-0FxjUpJuXVowThgf_Batch_2d94d28c-ccaa-3dd5-865c-faad50e542eb`.
+4. Turn on Container Insights for the Batch Compute Environment using the instructions found [here](https://docs.aws.amazon.com/batch/latest/userguide/cloudwatch-container-insights.html).
+5. Go to the "ECS" section of the AWS console
+6. Go to the "Clusters" [menu](https://us-east-1.console.aws.amazon.com/ecs/v2/clusters?region=us-east-1)
+7. Find the ECS cluster that starts with your Batch Computer Environment that you identified before. For example, in my case, the ECS cluster is called, `BatchTaskBatchComputeEnv-0FxjUpJuXVowThgf_Batch_2d94d28c-ccaa-3dd5-865c-faad50e542eb`.
+
+
+### You have now set up `tooltesterConfig.yml`
+It should look something like this,
+```
+SERVER-URL: https://qa.dockstore.org/api
+TOKEN: 000000000000000000000000000000000000000000
+AWS-AUTHORIZATION: fhembroff
+WDL-WES-URL: https://example1.execute-api.us-east-1.amazonaws.com/prod/ga4gh/wes/v1
+CWL-WES-URL: https://example2.execute-api.us-east-1.amazonaws.com/prod/ga4gh/wes/v1
+WDL-ECS-CLUSTER: BatchTaskBatchComputeEnv-0000000000000000_Batch_00000000-0000-0000-0000-000000000000
+CWL-ECS-CLUSTER: BatchTaskBatchComputeEnv-1111111111111111_Batch_11111111-1111-1111-1111-111111111111
+```
+
+</details>
 
 ### Workflows
-In order to run this command, you of course need workflows to run it on. You can add workflows for the `run-workflows` command to run [here](https://github.com/dockstore/dockstore-support/blob/8775ecadaec8b5fa36d03bc6ba962509ea38ca29/tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowList.java#L48-L55).
+In order to run this command, you of course need workflows to run it on. You can add workflows for the `run-workflows-through-wes` command to run [here](https://github.com/dockstore/dockstore-support/blob/8775ecadaec8b5fa36d03bc6ba962509ea38ca29/tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowList.java#L48-L55).
```

Comment:
As in other comments, I would avoid linking to specific commits. If somebody later adds and/or removes a workflow on the main branch, this link wouldn't show it.